src/charm.py

```diff
@@ -32,6 +32,11 @@
 from ops.pebble import Layer
 
 from charm_config import CharmConfig, CharmConfigInvalidError, CNIType
+from du_parameters.absolute_freq_ssb import get_absolute_frequency_ssb
+from du_parameters.carrier_bandwidth import get_carrier_bandwidth
+from du_parameters.dl_absolute_freq_point_a import get_dl_absolute_frequency_point_a
+from du_parameters.frequency import ARFCN, Frequency
+from du_parameters.initial_bwp import get_initial_bwp
```

Comment:
One way of simplifying the imports would be to add a `__init__.py` file to the `du_parameters` directory, and import the public names from the other files there.

Then we import everything:

```suggestion
from du_parameters import (
    get_absolute_frequency_ssb,
    get_carrier_bandwidth,
    get_dl_absolute_frequency_point_a,
    ARFCN,
    Frequency,
    get_initial_bwp,
)
```

Reply:
Done